themes/jb/layouts/partials/people/smallest.html

```diff
@@ -2,7 +2,7 @@
   <a href="/{{ .Type | pluralize }}/{{ .Params.username }}">
     <figure class="image is-96x96" style="margin: 0 auto">
     <!-- <figure class="image is-96x96 my-0 mx-auto"> -->
-      <img src="{{ .Params.avatar }}" alt="{{ .Title }}" class="is-rounded" />
+      <img src="{{ .Params.avatar }}" alt="{{ .Title }}" class="custom__set-1by1 is-rounded" />
```

Comment:
Would the Bulma `is-square` or `is-1by1` work for this? Or is the `object-fit: cover` not included in that?
https://bulma.io/documentation/elements/image/#responsive-images-with-ratios

Reply:
Unfortunately no, I did try those first.

Reply:
is-square
![image](https://user-images.githubusercontent.com/11327907/189221887-f4568fee-7688-447e-a842-ff126f67065d.png)

is-1by1
![image](https://user-images.githubusercontent.com/11327907/189222004-026ae1f4-d379-42e8-98f9-795ef52b56fe.png)

Reply:
nice trial&error ; )

Reply:
More like attempt to use the "supported" styles/classes before adding custom parts.

Reply:
that gets me thinking then - should we use the "supported" methods differently to create all of the thumbnails?

As in: is this a band-aid, or should we be revisiting how all the thumbnails get generated?

Reply:
This will provide greater coverage should an image not meet a 1:1 aspect ratio. I'm not exactly sure where they are pulled from or if JB can exert full control over them.

Reply:
most of the images were scraped by the scraper from Fireside. The images then were necessarily ingested by Fireside's process and likely made to conform to a specific standard.

Alex' photo is the first to be replaced independent of Fireside - he applied it here as a PR - and thus is the first case of integrating non-standard photos into our site here.

THAT is the difference, and why all a sudden Alex' photo is problematic.

Thus likely we will need to test a variety of images to see how Hugo & company manage their integration seamlessly.

Reply:
Kind of tangentially related, talked about images sizing here a bit: https://github.com/JupiterBroadcasting/jupiterbroadcasting.com/issues/326#issuecomment-1224064063